package.json

```diff
@@ -1,39 +1,20 @@
 {
-  "name": "eegi-web-app",
+  "name": "eleuther",
   "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^5.11.4",
-    "@testing-library/react": "^11.1.0",
-    "@testing-library/user-event": "^12.1.10",
-    "is-svg": "^4.3.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-scripts": "4.0.3",
-    "web-vitals": "^1.0.1"
-  },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "dev": "next dev",
```

Comment:
this is a minor thing but a lot of projects that people use to start with the default port of 3000, and in that case, the app won't run if something is already running there. Maybe you can just write it to sue different port like 3005

"dev": "next dev -p 3005"

Reply:
This isn't going to be SSR (server side rendering) so no need for server port at all. dev is development, which is testing. When published on Netlify, there's no need to use ports.